cupy/linalg/norms.py

```diff
@@ -22,20 +22,17 @@ def norm(x, ord=None, axis=None, keepdims=False):
         cupy.ndarray
 
     """
-    if not issubclass(x.dtype.type, (numpy.inexact, numpy.object_)):
+    if not issubclass(x.dtype.type, numpy.inexact):
```

Comment:
https://docs.scipy.org/doc/numpy/_images/dtype-hierarchy.png
Maybe does not need to check `numpy.object_`.